pkgs/applications/altcoins/exodus/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, lib, makeWrapper, fetchurl, unzip, glib, systemd, nss, nspr, gtk3-x11, gnome2, atk, cairo, gdk_pixbuf, xorg, xorg_sys_opengl, utillinux, alsaLib, dbus, at-spi2-atk, cups, vivaldi-ffmpeg-codecs, libpulseaudio }:
+
+stdenv.mkDerivation rec {
+  pname = "exodus";
+  version = "19.5.24";
+
+  src = fetchurl {
+    url = "https://exodusbin.azureedge.net/releases/${pname}-linux-x64-${version}.zip";
+    sha256 = "1yx296i525qmpqh8f2vax7igffg826nr8cyq1l0if35374bdsqdw";
+  };
+
+  sourceRoot = ".";
+  unpackCmd = ''${unzip}/bin/unzip "$src" -x "Exodus*/lib*so"'';
+
+  installPhase = ''
+	mkdir -p $out/bin $out/share/applications
+	cd Exodus-linux-x64
+	cp -r . $out
+	ln -s $out/Exodus $out/bin/Exodus
+	ln -s $out/exodus.desktop $out/share/applications
+	substituteInPlace $out/share/applications/exodus.desktop --replace 'Exec=bash -c "cd `dirname %k` && ./Exodus"' "Exec=Exodus"
```

Comment:
Maybe split this command onto multiple lines like the patchelf one below

Reply:
Will do.